python/appledoublekaitai/testdata/result/_Keynote2013-Sample09.key.apple_double_kaitai_plugin.0.text

```diff
@@ -0,0 +1,65 @@
+{
+  "magic": {
+    "name": "APPLE_DOUBLE",
+    "value": 333319
+  },
+  "version": 131072,
+  "reserved": [
+    77,
```

Comment:
This clearly is ASCII/UTF-8 text, perhaps show as text?

Reply:
I have seen _ files that contain timestamps and URL's. Could you try to add _ files from the following experiment(s):

Log into google drive with default OS X browser, download a file and save into FAT file-system.  
Log into google drive with default OS X, download a file and save into Downloads and copy into FAT file-system
Log into google drive with default OS X, download a file and save into Downloads and move into FAT file-system


I notice a lot of readable text is not visible in the output:

```
└─>>> hd _Keynote2013-Sample09.key
00000000  00 05 16 07 00 02 00 00  4d 61 63 20 4f 53 20 58  |........Mac OS X|
00000010  20 20 20 20 20 20 20 20  00 02 00 00 00 09 00 00  |        ........|
00000020  00 32 00 00 00 7f 00 00  00 02 00 00 00 b1 00 00  |.2..............|
00000030  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00000050  00 00 00 00 41 54 54 52  00 00 00 00 00 00 00 b1  |....ATTR........|
00000060  00 00 00 98 00 00 00 19  00 00 00 00 00 00 00 00  |................|
00000070  00 00 00 00 00 00 00 01  00 00 00 98 00 00 00 19  |................|
00000080  00 00 15 63 6f 6d 2e 61  70 70 6c 65 2e 71 75 61  |...com.apple.qua|
00000090  72 61 6e 74 69 6e 65 00  71 2f 30 30 38 32 3b 35  |rantine.q/0082;5|
000000a0  64 64 37 35 38 64 63 3b  4b 65 79 6e 6f 74 65 3b  |dd758dc;Keynote;|
000000b0  00                                                |.|
000000b1
```

As far as I know  0x5dd758dc = 1574394076 = 2019-11-22T04:41:16+01:00

Reply:
You definitely raise an interesting point here. Parsing this to a readable text could be done as a separate deliverable in the Kaitai plugin epic. It is not part of the minimum expected deliverable. It won't be done in this PR. 

You could bring this to the attention of the requester @Harm-van-Beek or our PO @remco-nfi if you want this implemented.

Reply:
You can use xattr to check the attributes in the way they are visualised by OS-X

https://gist.github.com/coisme/f89b886ebc508a64fae9b5e2bafb27b5

Reply:
@Schramp thanks for your contribution!

A ticket that addresses improved handling of byte arrays resulting from kaitai struct parsing is made and according to planning, it should be done this week.

see HANSKEN-20958 in jira